tinydb/storages.py

```diff
@@ -88,7 +90,7 @@ def __init__(self, path, create_dirs=False, **kwargs):
         super(JSONStorage, self).__init__()
         touch(path, create_dirs=create_dirs)  # Create file if not exists
         self.kwargs = kwargs
-        self._handle = open(path, 'r+')
+        self._handle = io.open(path, 'r+', encoding=encoding)
```

Comment:
I think if you use [`codecs.open`](https://docs.python.org/2/library/codecs.html#codecs.open), you don't have to use `serialized = unicode(serialized)` below, which would make it cleaner in my eyes. Could you test if that works?